src/managers/GuildEmojiRoleManager.js

```diff
@@ -72,7 +72,10 @@ class GuildEmojiRoleManager extends DataManager {
       resolvedRoleIds.push(roleId);
     }
 
-    const newRoles = this.cache.keyArray().filter(id => !resolvedRoleIds.includes(id));
+    const newRoles = this.cache
+      .keys()
```

Comment:
Wait... isn't this.cache a collection? keys is an iterator, not a method that returns all keys in an array form

Reply:
I'm kinda suprised I didn't crash while testing this tbh